commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -393,12 +398,43 @@ def update_return_payment_state_after_successful_refund(self, order_id: str,
                 return_item_id=return_line_item_return_id,
                 payment_state=ReturnPaymentState.REFUNDED
             )
+            if not payment_intent_id:
+                payment_intent_id = ''
+            logger.info(f'Creating return for order - payment_intent_id: {payment_intent_id}')
+
+            payment = self.get_payment_by_key(payment_intent_id)
+            logger.info(f"Payment found: {payment}")
+            order = self.get_order_by_id(order_id=order_id)
+            transaction_id = find_refund_transaction(order, amount_in_cents)
+            update_transaction_id_action = OrderSetReturnItemCustomTypeAction(
+                return_item_id=return_line_item_return_id,
+                type=CTTypeResourceIdentifier(
+                    key='returnItemCustomType',
+                ),
+                fields=CTFieldContainer({
+                    'transactionId': transaction_id
+                })
+            )
+            return_transaction_return_item_action = PaymentSetTransactionCustomTypeAction(
+                transaction_id=transaction_id,
+                type=CTTypeResourceIdentifier(key='transactionCustomType'),
+                fields=CTFieldContainer({
+                    'returnItemId': return_line_item_return_id
+                })
+            )
+            logger.info(f"Update return payment state after successful refund - payment_intent_id: {payment_intent_id}")
 
             updated_order = self.base_client.orders.update_by_id(
                 id=order_id,
                 version=order_version,
-                actions=[return_payment_state_action]
+                actions=[return_payment_state_action, update_transaction_id_action]
+            )
+            _ = self.base_client.payments.update_by_id(
```

Comment:
`_ =` isn't required. If you are not doing anything with the return value, you should be okay to not assign it to a variable at all